src/data/items.js

```diff
@@ -99,6 +100,32 @@ export const carrot = crop({
   name: 'Carrot',
 })
 
+/**
+ * @property farmhand.module:items.asparagusSeed
+ * @type {farmhand.item}
+ */
+export const asparagusSeed = crop({
+  cropType: ASPARAGUS,
+  cropTimetable: {
+    [SEED]: 4,
+    [SPROUT]: 1,
+    [GROWING]: 5,
+  },
+  growsInto: 'asparagus',
+  id: 'asparagus-seed',
+  name: 'Asparagus Seed',
+  tier: 1,
```

Comment:
Let's make this tier 4 (and move it below the other crops in this file). Similarly to what I mentioned in [this comment](https://github.com/jeremyckahn/farmhand/issues/104#issuecomment-855300761), I'd like each tier to only have three crops.

Reply:
updated